### ISSUE TITLE
AWS Errors during backup

### ISSUE THREAD

**Issue by vdrover** [watchfulli/XCloner-Wordpress#132]
On the Watchful blog, the backup is working via CLI, but the following errors are reported:

```
PHP Warning:  Invalid argument supplied for foreach() in /home/abcxyz/webapps/retail/public/blog/wp-content/plugins/xcloner-backup-and-restore/vendor/aws/aws-sdk-php/src/Endpoint/PartitionEndpointProvider.php on line 116
PHP Warning:  Invalid argument supplied for foreach() in /home/abcxyz/webapps/retail/public/blog/wp-content/plugins/xcloner-backup-and-restore/vendor/aws/aws-sdk-php/src/Endpoint/PartitionEndpointProvider.php on line 116
PHP Warning:  Invalid argument supplied for foreach() in /home/abcxyz/webapps/retail/public/blog/wp-content/plugins/xcloner-backup-and-restore/vendor/aws/aws-sdk-php/src/Endpoint/PartitionEndpointProvider.php on line 116
```

**Comment by ovidiul** [comment]
I've posted a fix in the milestone 4.2.13 branch, please feel free to get reinstall the package and let me know if you have anymore issues.

**Comment by vdrover** [comment]
Fix confirmed.
